x-pack/plugins/security_solution/common/siem_migrations/model/api/rules/rule_migration.schema.yaml

```diff
@@ -79,6 +79,70 @@ paths:
                     type: boolean
                     description: Indicates rules migrations have been updated.
 
+  /internal/siem_migrations/rules/install:
```

Comment:
These routes only affect one migration, if we are consistent with the rest of the routes we should put them under {{migration_id}} path:
- `/internal/siem_migrations/rules/{migration_id}/install`
- `/internal/siem_migrations/rules/{migration_id}/install_all`

Reply:
done